src/components/Tool/Appeal/AddAppealForm.tsx

```diff
@@ -105,7 +154,32 @@ const AddAppealForm = (): ReactElement => {
     name: yup.string().required('Please enter a name'),
   });
 
-  return (
+  const contactStatusesFormatted = Object.entries(
+    contactTags,
+  ).map(([tag, translatedTag]) => ({ value: tag, title: translatedTag }));
```

Comment:
I think these can be loaded from the API. Since they are loaded from the API in the current app, we should probably keep doing that. (Also they are statuses, not tags 😄 your variable names are misleading).

You could reuse [ContactFilters.graphql](https://github.com/CruGlobal/mpdx-react/blob/e4ac3bbb40b0c0e27d8918dc3ce9df4be55888d2/src/components/Contacts/ContactFilters/ContactFilters.graphql) or make a new one that specified fewer fields. There's currently no way to only ask for certain filters. Maybe we could get that added but it's probably not too much overfetching to just use it as is. Here's my stab at a slightly simpler one but it's not that many fewer fields...
<img width="814" alt="Screen Shot 2021-10-01 at 4 49 13 PM" src="https://user-images.githubusercontent.com/756501/135696565-8892d31e-27af-4fec-87d7-acf3f7f0d565.png">